### DISCUSSION TITLE
Which plugin manager should I use for ZSH

### DISCUSSION THREAD

**Discussion by muuvmuuv** [starship/starship#2256] [Q&A] [answered]
So since Antibody is deprecated I wonder which zsh plugin managers you all use and which one is recommended.

**Comment by matchai** [comment]
I'm a fan of [zplug](https://github.com/zplug/zplug). Lightweight, fast, and easy!

**Reply by muuvmuuv** [reply]
Will try it out, thank you! Maybe Starship will ship with some basic plugin manager someday ^^

**Comment by cappyzawa** [comment, marked as answer]
I use [zdharma/zinit: Flexible and fast Zsh plugin manager with clean fpath, reports, completion management, Turbo, annexes, services, packages\.](https://github.com/zdharma/zinit).

And, the starship is installed as follows.
```
zinit ice lucid as"program" from"gh-r" bpick"starship-x86_64-*.tar.gz" \
  atclone'starship completions > ~/.zsh/Completion/_starship' atpull'%atclone' \
  atload'export STARSHIP_CONFIG=$XDG_CONFIG_HOME/starship/starship.toml'
zinit light starship/starship
```

**Reply by muuvmuuv** [reply]
Thanks! I would manually source it after Starship. Are there any benefits by loading Starship through zinit?

**Reply by muuvmuuv** [reply]
Okay I tested both and holy hell is zinit fast. I bought the new Mac mini M1 and decided to go fresh with my setup. iTerm with Starship, pyenv, rbenv, volta and zinit + plugins starts in 150ms. This is amazing.
